### ISSUE TITLE
Add a 915MHz antenna design

### ISSUE THREAD

**Issue by SloMusti** [IRNAS/ttn-irnas-gw#2]
Add a 915MHz antenna design for users in the USA.

**Comment by RussNelson** [comment]
Looking at the precision necessary for the loops, would it make sense to create a 3D printed part around which you could simply wrap the wire and know it would have the correct geometry?

**Comment by RussNelson** [comment]
1.4mm wire is 15 AWG: https://www.amazon.com/Remington-Industries-15SNSP-5-Enameled-Diameter/dp/B01BDB7ML4/ref=sr_1_26?ie=UTF8&qid=1525920217&sr=8-26&keywords=15+awg+wire

**Comment by RussNelson** [comment]
Is this the correct RP SMA Male to use? https://www.amazon.com/uxcell-15pcs-Solder-Adapter-Connector/dp/B01ES9ULRA/ref=sr_1_1?ie=UTF8&qid=1525920591&sr=8-1&keywords=uxcell+15pcs+rp+sma+male+jack+solder+pcb+clip+edge

**Comment by RussNelson** [comment]
The video says that it's an 868/915 MHz antenna. Or is there something different between the two different frequencies? They're 5% different, so I would think that the measurements for the antenna would be 5% between them.

**Comment by SloMusti** [comment]
[Co-linear antenna modeling_915MHz.pdf](https://github.com/IRNAS/ttn-irnas-gw/files/1997495/Co-linear.antenna.modeling_915MHz.pdf) The 915MHz design is different, we have not tested it yet but will check and post instructions and details.

**Comment by RussNelson** [comment]
Thanks. Parts ordered. Will start working on the 3D printed part. Silly question: if the part is designed to stay in the loop, will that affect the RF response?

**Comment by SloMusti** [comment]
@RussNelson Yes, any material that is not air in the loop will change the properties of the antenna, however the effects may be.sufficiently small for this to still work.

**Comment by RussNelson** [comment]
Just bent it around a suitable-diameter rod. Need to get the right connector in the mail before I can try it out.

![img_20180521_162412](https://user-images.githubusercontent.com/1156994/40330052-ba1ec916-5d19-11e8-81f2-e63afa5415aa.jpg)

**Comment by heyitsyang** [comment]
@RussNelson, Did you ever get a 3D part printed for the loops on the antenna?  How did it work out?

**Comment by RussNelson** [comment]
No, I bent it using a metal rod.

**Comment by happytm** [comment]
@RussNelson did you ever test this design for 915 mhz ? If so how was the performance  ?

Thanks.
